### DISCUSSION TITLE
Price

### DISCUSSION THREAD

**Discussion by SilvioTrombin** [zepp-health/zeppos-forum#377] [General]
Hello everybody, plaese, can you tell me how to upload my watchfaces with a price? Thanks in advance

**Comment by MyoungXUE** [comment]
Sorry, uploading paid watch faces to the official store is currently not supported.

**Reply by jordhewson** [reply]
There are a lot of paid watch faces in the official store (at least in the US). So it must be possible, if only in some regions.

**Reply by MyoungXUE** [reply]
As you say. The Zepp Health company have strict limitation on paid apps. You can find more informations in this document.

https://docs.zepp.com/docs/guides/faq/paid-app/
